features/step_definitions/cli.rb

```diff
@@ -23,6 +23,23 @@
   end
 end
 
+When /^I run the :([a-z_]*?)( background)? client command with importmode if supported with:$/ do |yaml_key, background, table|
+  opts = table.raw
+  if env.version_ge("4.13", user: user)
```

Comment:
@lwan-wanglin Hi, `new-app` and `new-build` add importMode option since 4.14, not support in 4.13